packages/shared/src/components/squads/SquadHeaderBar.tsx

```diff
@@ -55,43 +53,34 @@ export function SquadHeaderBar({
     <div
       {...props}
       className={classNames(
-        'flex h-fit w-full flex-row justify-center gap-4 tablet:w-auto',
+        'no-scrollbar flex h-fit w-full flex-row justify-center gap-4 overflow-x-auto pr-4 tablet:w-auto tablet:pr-0',
         className,
       )}
     >
-      <div className="relative">
-        {verifyPermission(squad, SourcePermissions.Invite) &&
-          !showJoinButton && (
-            <Button
-              variant={ButtonVariant.Secondary}
-              className={classNames(
-                tourIndex === TourScreenIndex.CopyInvitation &&
-                  'highlight-pulse',
-              )}
-              onClick={() => {
-                trackAndCopyLink();
-              }}
-              icon={<AddUserIcon />}
-              disabled={copying}
-            >
-              Invitation link
-            </Button>
+      {verifyPermission(squad, SourcePermissions.Invite) && !showJoinButton && (
+        <Button
+          variant={ButtonVariant.Secondary}
+          className={classNames(
+            'ml-14 tablet:ml-0',
+            tourIndex === TourScreenIndex.CopyInvitation && 'highlight-pulse',
           )}
-      </div>
+          onClick={() => {
+            trackAndCopyLink();
+          }}
+          icon={<AddUserIcon />}
+          disabled={copying}
+        >
+          Invitation link
+        </Button>
+      )}
       {showJoinButton && (
         <SquadJoinButton
           className="flex w-full flex-1 tablet:ml-auto tablet:w-auto tablet:flex-initial"
           squad={squad}
           origin={Origin.SquadPage}
         />
       )}
-      {sidebarRendered && (
-        <SquadMemberShortList
-          squad={squad}
-          members={members}
-          className="hidden laptopL:flex"
-        />
-      )}
+      <SquadMemberShortList squad={squad} members={members} />
```

Comment:
So this will no longer be dependant on the sidebar?

Reply:
Yeah, that was only to know whether it is touch device or not.